internal/pkg/cli/job_delete.go

```diff
@@ -159,6 +230,111 @@ func (o *deleteJobOpts) askJobName() error {
 	return nil
 }
 
+func (o *deleteJobOpts) appEnvironments() ([]*config.Environment, error) {
+	envs := o.environments
```

Comment:
is this needed? (and can we remove the `environments` field from the struct)

Reply:
Done.